docs/maps/Collection.json

```diff
@@ -125,6 +125,13 @@
           "description": "Administrative or Internal project this resource is a part of.",
           "type": "string"
         },
+        "releaseTags": {
+          "description": "Tags for release",
```

Comment:
A tad more description would be helpful. I have no idea what a release tag is.

Reply:
@justinlittman that makes two of us.  This currently is an XML node that gets made in identityMetadata when you press the "Manage Release" button in Argo.  I've asked @andrewjbtw to help explain it more.